cli/src/potpie/utility.py

```
@@ -0,0 +1,63 @@
+import os
+import logging
```

Ruff (0.8.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)